react-charts/donut-utilization-chart/step7.md

```diff
@@ -26,21 +26,49 @@ Now we'll change the donut utilization chart theme color...
     right: 290,
     top: 20
   }}
+  themeColor={ChartThemeColor.green}
   thresholds={[{ value: 60 }, { value: 90 }]}
   height={230}
   width={350}
 /&gt;
 </pre>
 
-3) Add the following property to that section:
+3) <strong>Refactor `height`, `padding`, `themeColor`, and `width` properties from `ChartDonutUtilization` to the `ChartDonutThreshold` component.</strong>
+
+ The `ChartDonutThreshold` component is a visual representation of donut utilization thresholds.  The `constrainToVisibleArea` property will be reintroduced in the next step.

```

Comment:
The user doesn't know what the `constrainToVisibleArea` property does yet, so we can just remove that line.

```suggestion
 The `ChartDonutThreshold` component is a visual representation of donut utilization thresholds.
```